app/src/main/java/de/westnordost/streetcomplete/quests/AbstractQuestAnswerFragment.kt

```diff
@@ -324,6 +343,10 @@ abstract class AbstractQuestAnswerFragment<T> :
         }
     }
 
+    private fun onClickConvertToStepsAnswer() {
+        listener?.onConvertToSteps(questKey as OsmQuestKey)
```

Comment:
Should there be a warning first along the lines of "Please split the way accordingly before converting the whole way into steps."

Reply:
It's not really done for other quests, is it? Hmm. @matkoniecz suggested that. Is this based on any user tests / known issues? 

Or rephrase other answer option to "Whole way are steps"?

Reply:
> Whole way are steps

+1, though is it maybe rather

> Whole way is steps

(way is singular here, not plural)

Reply:
What about my suggestion here?
https://github.com/streetcomplete/StreetComplete/pull/3334#discussion_r720684849